Validation/RecoEgamma/plugins/ElectronMcFakePostValidator.cc

```diff
@@ -59,6 +59,14 @@ void ElectronMcFakePostValidator::finalize( DQMStore::IBooker & iBooker, DQMStor
     h1_ele_xOverX0VsEta->setBinContent(ibin,xOverX0) ;
   }
 /**/
+
+  MonitorElement * h1_ele_provenance = get(iGetter, "provenance") ;
+  h1_ele_provenance->getTH1F()->Scale(1./h1_ele_provenance->getBinContent(3));
+  MonitorElement * h1_ele_provenance_barrel = get(iGetter, "provenance_barrel") ;
+  h1_ele_provenance_barrel->getTH1F()->Scale(1./h1_ele_provenance_barrel->getBinContent(3));
+  MonitorElement * h1_ele_provenance_endcaps = get(iGetter, "provenance_endcaps") ;
+  h1_ele_provenance_endcaps->getTH1F()->Scale(1./h1_ele_provenance_endcaps->getBinContent(3));
```

Comment:
Hi, I made corrections on the files on Provenance_Histos_Renormalization_81X_V1 #16246
Do I have to made another PR ?